### DISCUSSION TITLE
how is it possible that I can download the snap version from github and without using snap store?

### DISCUSSION THREAD

**Discussion by ghost** [flameshot-org/flameshot#2085] [Q&A]
is it possible to distribute apps in snap format without using the canonical snap store?

**Comment by mmahmoudian** [comment]
This question is slightly off topic, but I hope these links help you:

- https://askubuntu.com/questions/1266894/how-can-i-install-a-snap-package-from-a-local-file
- https://forum.snapcraft.io/t/offline-snap-installers-and-possibility-to-update/275
